lib/features/session/presentation/utils/session_utils.dart

```diff
@@ -0,0 +1,17 @@
+
+import 'package:core/utils/app_logger.dart';
+
+class SessionUtils {
+  static Uri refactorApiUriForJMapDartClient(Uri apiUri) {
+    final pathSegments = apiUri.pathSegments.where((segment) => segment.isNotEmpty);
+    log('SessionUtils::refactorApiUriForJMap(): pathSegments: $pathSegments');
+    if (pathSegments.isNotEmpty && pathSegments.last == 'jmap') {
```

Comment:
IMO, it is ok for the case of `fastmail` and `tmail` (jmap is at the end of api uri). But we not sure all the server follow that. WDYT?

Reply:
Oke. We will remove path segment `jmap` of `baseUrl` in `jmap_dart_client` to sync with TMail. IMO, it will compatibility all server.